src/conv-core/memory-isomalloc.C

```diff
@@ -85,21 +80,21 @@ static void *meta_malloc(size_t size)
         int _isomalloc_thread = isomalloc_thread;
         if (CmiThreadIs(CMI_THREAD_IS_TLS)) _isomalloc_thread = 1;
 #endif
-	if (meta_inited && CpvInitialized(isomalloc_blocklist) && CpvAccess(isomalloc_blocklist)
+	if (meta_inited && CpvInitialized(isomalloc_state) && CpvAccess(isomalloc_state).context && !CpvAccess(isomalloc_state).disabled
```

Comment:
Could pull this long if-statement out into a function returning a bool. It's repeated below

Reply:
Done